tests/paz/backend/render_test.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 import numpy as np
 
-from paz.backend.render import calculate_norm
+from paz.backend.standard import calculate_norm
```

Comment:
calculate_norm is transferred to the standard module. All the standard module test cases are in backend/standard_test.py. Here the calculate_norm is imported because of the dependencies.